build.sbt

```diff
@@ -27,7 +27,7 @@ lazy val commonSettings =
     Seq(
       organization := "fr.davit",
       organizationName := "Michel Davit",
-      version := "1.2.1-SNAPSHOT",
+      version := "1.2.2-SNAPSHOT",
```

Comment:
`1.2.1-SNAPSHOT` is a pre-release version. `1.2.1` has not been released.
Since your changes include a new feature, it would make sense to bump the major version instead to `1.3.0-SNAPSHOT`